include/rest-request.php

```diff
@@ -8,20 +8,61 @@
  * accessible as $polylang global object
  *
  * Properties:
- * options        => inherited, reference to Polylang options array
- * model          => inherited, reference to PLL_Model object
- * links_model    => inherited, reference to PLL_Links_Model object
- * links          => reference to PLL_Admin_Links object
- * static_pages   => reference to PLL_Static_Pages object
- * filters        => reference to PLL_Frontend_Filters object
- * filters_links  => reference to PLL_Filters_Links object
- * posts          => reference to PLL_CRUD_Posts object
- * terms          => reference to PLL_CRUD_Terms object
+ * options              => inherited, reference to Polylang options array
+ * model                => inherited, reference to PLL_Model object
+ * links_model          => inherited, reference to PLL_Links_Model object
+ * links                => reference to PLL_Admin_Links object
+ * static_pages         => reference to PLL_Static_Pages object
+ * filters              => reference to PLL_Frontend_Filters object
+ * filters_links        => reference to PLL_Filters_Links object
+ * filters_sanitization => reference to PLL_Filters_Sanitization object
+ * posts                => reference to PLL_CRUD_Posts object
+ * terms                => reference to PLL_CRUD_Terms object
  *
  * @since 2.6
  */
 class PLL_REST_Request extends PLL_Base {
-	public $links, $static_pages, $posts, $terms, $filters, $filters_links;
+	/**
+	 * Instance of PLL_Admin_Links
+	 *
+	 * @var PLL_Admin_Links
+	 */
+	public $links;
+
+	/**
+	 * Instance of PLL_Static_Pages
+	 *
+	 * @var PLL_Static_Pages
+	 */
+	public $static_pages;
+
+	/**
+	 * Instance of PLL_CRUD_Posts
+	 *
+	 * @var PLL_CRUD_Posts
+	 */
+	public $posts;
+
+	/**
+	 * Instance of PLL_CRUD_Terms
+	 *
+	 * @var PLL_CRUD_Terms
+	 */
+	public $terms;
+
+	/**
+	 * Instance of PLL_Frontend_Filters
+	 *
+	 * @var PLL_Frontend_Filters
+	 */
+	public $filters;
+
+	/**
+	 * Instance of PLL_Filters_Links
+	 *
+	 * @var PLL_Filters_Links
+	 */
+	public $filters_links;
```

Comment:
I don't believe that any modification to this file is necessary.

Reply:
Due to the fact ay the beginning of the PR we instantiated PLL_Filters_Sanitization class in this class.
I'm going to revert these modifications